apps/main.c

```diff
@@ -47,6 +47,34 @@ static twin_pixmap_t *load_background(twin_screen_t *screen, const char *path)
         twin_pixmap_destroy(raw_background);
         return twin_make_pattern();
     }
+#if defined(CONFIG_PIXMAN)
```

Comment:
Don't do that. The rendering path to Pixman should be transparent to all applications using Mado, meaning there should be no changes required on the application side when Mado is configured to switch from the built-in pixel manipulation to the faster Pixman path.